### PR TITLE
update spec

### PR DESCRIPTION
**Pull request by ingvagabund**
Can you try this one? Built with golang-1.5.3-1.fc24.x86_64

### CODE REVIEW

File: skopeo.spec [hunk 1]
```diff
@@ -204,7 +216,7 @@ sort -u -o devel.file-list devel.file-list
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/Godeps/_workspace:%{gopath}
+export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor/_workspace:%{gopath}
```
Comment (runcom) [inline skopeo.spec:219]: is `_workspace` right here? isn't it only for godeps?

**Comment by runcom** [comment]
just a question - I'll merge then

thanks a lot @ingvagabund :heart:

**Comment by runcom** [comment]
merged!

File: skopeo.spec (hunk 1 above)
Comment (ingvagabund) [inline skopeo.spec:219]: You are right. "/_workspace" must be deleted. Did not noticed that when rewritting the expression. As tests are not run, this has no effect on the final rpms.

**Pull request closed without merging**